Source/WebCore/Modules/modern-media-controls/controls/text-tracks.css

```diff
@@ -45,7 +45,7 @@ video::-webkit-media-text-track-container {
     -webkit-line-box-contain: block inline-box replaced;
 }
 
-video[controls]::-webkit-media-text-track-container.visible-controls-bar {
+video[controls][pseudo="-webkit-media-text-track-container"].visible-controls-bar {
```

Comment:
I just realized that the pseudo attribute won't be on the video element so this won't work. And we probably cannot pierce the shadow tree. Is there a selector to see whether the shadow host is a video element @nt1m?

Reply:
You can't just pierce the shadow tree like that.

However, if the stylesheet is appended inside the shadow tree, you should be able to do:

`:host([controls]) [pseudo="-webkit-media-text-track-container"].visible-controls-bar`